src/preamble_minimal.js

```diff
@@ -61,14 +62,15 @@ Module['wasm'] = base64Decode('<<< WASM_BINARY_DATA >>>');
 #include "runtime_functions.js"
 #include "runtime_strings.js"
 
-var HEAP8, HEAP16, HEAP32, HEAPU8, HEAPU16, HEAPU32, HEAPF32, HEAPF64;
+var _HEAP_DATA_VIEW, HEAP8, HEAP16, HEAP32, HEAPU8, HEAPU16, HEAPU32, HEAPF32, HEAPF64;
```

Comment:
Is there a reason this is prefixed with `_`?

Reply:
Changed to HEAP_DATA_VIEW